libs/application/templates/directorate-of-immigration/citizenship/src/utils/childrenInfo.ts

```
@@ -44,16 +44,18 @@ export const getSelectedCustodyChildren = (
 ): ApplicantChildCustodyInformation | undefined => {
   const custodyChildren = getValueViaPath(
     externalData,
     'childrenCustodyInformation.data',
     undefined,
   ) as ApplicantChildCustodyInformation[] | undefined
 
-  const childInfo = custodyChildren && custodyChildren[sectionIndex]
-  const childNationalId = childInfo?.nationalId
+  const childInfo = childNationalId
+    ? custodyChildren?.filter((x) => x.nationalId === childNationalId)[0]
+    : custodyChildren && custodyChildren[index]
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)